pkg/provisioner/provisioner.go

```diff
@@ -107,6 +107,15 @@ type PrepareData struct {
 	TargetFirmwareComponents []metal3api.FirmwareUpdate
 }
 
+type ServicingData struct {
+	FirmwareConfig              *metal3api.FirmwareConfig
+	TargetFirmwareSettings      metal3api.DesiredSettingsMap
+	ActualFirmwareSettings      metal3api.SettingsMap
+	LiveFirmwareUpdateAllowed   bool
+	LiveFirmwareSettingsAllowed bool
```

Comment:
These two fiels are not necessary since they are only used locally. You can just create two local boolean variables in checkServicing.

Reply:
Agreed, will apply this suggestion in the next revision.

Reply:
Not resolved, the variables should be deleted from here

Reply:
apologies, I must have accidentally added them back in while rebasing. Looks like they are gone now.